sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -852,7 +852,19 @@ void gnrc_ipv6_netif_init_by_dev(void)
             _add_addr_to_entry(ipv6_if, &addr, 64, 0);
 
         }
-
+#ifdef RIOT_STATIC_IPV6_LLADDR
+        /* parse addr from string and explicitely set a link lokal prefix
+         * if ifnum > 1 each interface will get its own link local address
+         * with RIOT_STATIC_IPV6_LLADDR + i
+         */
+        char addr_str[] = RIOT_STATIC_IPV6_LLADDR;
+        ipv6_addr_t lladdr;
+        if(ipv6_addr_from_str(&lladdr, addr_str) != NULL) {
+            ipv6_addr_set_link_local_prefix(&lladdr);
+            lladdr.u8[15] += i;
+            _add_addr_to_entry(ipv6_if, &lladdr, 64, 0);
+        }
```

Comment:
Yapp, was rereading the RFCs just now, and can confirm that there is nothing pointing to a problem, so alright.